src/main/resources/templates/index.html

```diff
@@ -11,9 +11,27 @@
 		<div th:replace="fragments/bootstrap_nav_header.html"></div>
 		<h2 class="text-center">Mapache Search</h2>
 		<div th:replace="fragments/basic_search_form.html"></div>
-		<div th:replace="fragments/bootstrap_footer.html"></div>
+		<div th:replace="fragments/bootstrap_scripts.html"></div>
+		
+		<!-- <div th:replace="fragments/bootstrap_footer.html"></div> -->
+		<!-- <div th:replace="fragments/bootstrap_scripts.html"></div> -->
     </div>
-	<div th:replace="fragments/bootstrap_scripts.html"></div>
-</body>
+	<h3 class="text-center">Recently Upvoted Link </h3>
+	<table class = "table">
+		<tbody>
+			<div th:if=" ${upVoteLinksSize} == 0">
+				<h4 class = "text-center"> No upvoted link</h4>
+			</div>
+			<div th:if=" ${upVoteLinksSize} != 0">
+				<tr th:each="f,iterStats: ${upVoteLinks}"> 
+					<div th:if="${iterStats.index} lt 5">
```

Comment:
I would rather put this logic inside the controller rather than inside the template